frontend/src/pages/NewRunV2.tsx

```diff
@@ -147,57 +143,8 @@ function NewRunV2(props: NewRunV2Props) {
 
   // Use pipeline, pipeline version, and experiment from parent component
   useEffect(() => {
-    setPipeline(existingPipeline);
-    setPipelineVersion(existingPipelineVersion);
     setApiExperiment(chosenExperiment);
```

Comment:
Question: Can we initialize apiExperiment as followed?
```
const [apiExperiment, setApiExperiment] = useState<ApiExperiment>(chosenExperiment);
```

Then we don't need to define `useEffect` here.

When user select an new experiment, the following line will make the state change:
https://github.com/kubeflow/pipelines/blob/fe28b08dfa29bedb7fd465344c177e7c5ee92821/frontend/src/pages/NewRunV2.tsx#L395